src/blockchain-utilities/derivation.ts

```diff
@@ -10,25 +10,19 @@ export const PROTOCOL_IDS = {
   AIRSEEKER: '5',
 };
 
-/**
- * An interface that reflects the structure of a Template
- */
-export interface Template {
-  airnode: string;
-  encodedParameters: string;
-  endpointId: string;
-}
-
 /**
  * Derives a template ID from the input parameters
  *
  * @param airnode an Airnode address
  * @param encodedParameters encoded parameters, see the airnode/abi package's encode function
  * @param endpointId An endpointID (see deriveEndpointId)
  */
-export const deriveTemplateId = ({ airnode, encodedParameters, endpointId }: Template) =>
+export const deriveTemplateIdV0 = (airnode: Address, encodedParameters: string, endpointId: string) =>
```

Comment:
Should endpointId be of type `Id` https://github.com/api3dao/commons/pull/62/files#diff-62ce94db571c980385663a7ea8012b9914db81e4a725ab3cf79ee89e5d9236fbR9?

Reply:
I'll improve the function types in https://github.com/api3dao/commons/pull/63